common/config/rush/variants/stable/pnpm-lock.yaml

```diff
@@ -542,7 +555,7 @@ packages:
     resolution: {integrity: sha512-QkBXwGgaoC2GtGZRoma6kv7Szfv06khvhFav67ZExau2RaXzy8MpHSMO2PNoP2XtmQphJQRHFfg77Bq731Yizw==}
     engines: {node: '>=6.9.0'}
     dependencies:
-      '@babel/types': 7.24.5
+      '@babel/types': 7.24.0
```

Comment:
Probably don't want a downgrade here?

Reply:
you would be right about these downgrades. having to merge main into this guy has been tricky managing this stuff will update all the down grades